Plugins/ProjectTemplatePlugin/ProjectDescriptionHelpers/InfoPlist/Project+InfoPlist.swift

```diff
@@ -38,6 +38,10 @@ extension InfoPlist {
       .setUISupportedInterfaceOrientations(["UIInterfaceOrientationPortrait"])
       .setCustomValue("baseURL", "$(BASE_URL)")
       .setCustomValue("chatBaseURL", "$(CHAT_BASE_URL)")
+      .setCustomValue("KAKAO_NATIVE_APP_KEY", "$(KAKAO_NATIVE_APP_KEY)")
+      .setCFBundleURLTypes("kakao$(KAKAO_NATIVE_APP_KEY)")
+      .setLSApplicationQueriesSchemes() // kakao
```

Comment:
```suggestion
      .setLSApplicationQueriesSchemes(["kakaokompassauth", "kakaolink"]) // kakao
```

혹시 이 부분에서 위와 같이 메서드 호출 시 아규먼트를 직접 전달하지 않은 이유가 있을까요!?